process/block/preprocess/transactions.go

```diff
@@ -383,16 +384,15 @@ func (txs *transactions) getAllTxsFromMiniBlock(
 //TODO move this constant to txFeeHandler
 const minGasLimitForTx = uint64(5)
 
-const numZerosForSCAddress = 10
-
 //TODO move this to smart contract address calculation component
 func isSmartContractAddress(rcvAddress []byte) bool {
 	isEmptyAddress := bytes.Equal(rcvAddress, make([]byte, len(rcvAddress)))
 	if isEmptyAddress {
 		return true
 	}
 
-	isSCAddress := bytes.Equal(rcvAddress[:(numZerosForSCAddress - 1)], make([]byte, numZerosForSCAddress-1))
+	isSCAddress := bytes.Equal(rcvAddress[:(hooks.NumInitCharactersForScAddress-hooks.VMTypeLen)],
```

Comment:
Could extract "hooks.NumInitCharactersForScAddress-hooks.VMTypeLen" in a separate variable